nifi-nar-bundles/nifi-snowflake-bundle/nifi-snowflake-services-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -29,3 +29,15 @@ The following binary components are provided under the Apache Software License v
     The following NOTICE information applies:
       snowflake-jdbc
       Copyright (c) 2013-2018 Snowflake Computing, Inc.
+
```

Comment:
I think `snowflake-jdbc` can be removed from the notice file because because we are depend on `snowflake-ingest-sdk`.
Also from the assembly notice file.